surface/include/pcl/surface/convex_hull.h

```diff
@@ -255,6 +263,9 @@ namespace pcl
       /* \brief z-axis - for checking valid projections. */
       const Eigen::Vector3d z_axis_;
 
+      /* \brief vector containing the point cloud indices of the convex hull points. */
+      std::vector<uint32_t> hull_indices_;
```

Comment:
From my point of view PCL is there not 100% consistent when indices are concerned. From my point of view it doesn't make sense to store data that is unsigned in a signed data type and the indices themselves are returned via `pcl::PointIndices`, so there shouldn't be a problem. Of course I can also change the internally used data type `int`, if that would be considered to be more consistent with the rest of PCL.